firstmoversadvantage/fma-ruby-on-rails-template
# 169735612 create contact us mvc

#### What's this PR do?
- Create a "Contact Us" feature that saves the contact requests in a database

#### How should this be manually tested?
1.
- Go to http://test.fmadata.com:3000/ in order to properly use hCaptcha
- Click `Contact us` link and try to send new contact request
2. 
- Create admin user (is_admin: true) 
- Login as admin
- Try to see contact requests on `/contact_requests` path

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/169735612

#### Task completed checklist:

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
